hydrolib/core/dflowfm/ini/util.py

```diff
@@ -622,3 +623,48 @@ def rename_keys_for_backwards_compatibility(
                 break
 
     return values
+
+
+class UnknownKeywordErrorManager:
+    """
+    Error manager for unknown keys.
+    Detects unknown keys and manages the Error to the user.
+    """
+
+    def raise_error_for_unknown_keywords(
+        self,
+        data: Dict[str, Any],
+        section_header: str,
+        fields: Dict[str, Any],
+        excluded_fields: Set,
+    ):
+        """
+        Notify the user of unknown keywords.
+
+        Args:
+            data (Dict[str, Any])   : Input data containing all set properties which are checked on unknown keywords.
```

Comment:
What are `set properties`? It's a dict, not a set? 😊

Reply:
will adjust